src/EditLabelDialog/EditLabelDialog.stories.jsx

```diff
@@ -0,0 +1,67 @@
+// write a story
+
+import { Button } from "@mui/material";
+import EditLabelDialog from "./EditLabelDialog";
+import React from "react";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  component: EditLabelDialog,
+  title: "Dialog/EditLabelDialog"
+};
+
+const Template = args => {
+  const [open, setOpen] = React.useState(false);
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+  const handleCancel = args => {
+    setOpen(false);
+    action("cancel")(args);
+  };
+  const handleSave = args => {
+    setOpen(false);
+    action("save")(args);
```

Comment:
`action` is intended to reflect the function name that we are using. In this case `onClose`, `onEdit`, `onNew`. Currently you are showing `cancel` and `save`.